### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by kuritka** [invopop/yaml#3] [General]
Hi @samlown , awsome job! It would be good to [declare it "an official yaml repo".](https://github.com/ghodss/yaml/issues/80) In such case, don't you think it would be good to provide `v1.0.0` and declare no breaking changes?

**Comment by samlown** [comment]
Hi @kuritka! Thanks! I'm just glad to see it working and remove the github security warnings :-)

I'll post an issue to the ghodss/yaml repo declaring this new fork more prominently.

Regarding v1 status. Given this is a fork, I'd feel more comfortable leaving in the v0 status for a bit longer to open the door up to possible additions or alternations that might make sense. Let the dust settle, so to speak. The API however is so simple that I can't foresee any major changes. Perhaps I'd consider fixing some of the linter complaints around the jarring `yaml.YAMLToJSON` type methods, but wrappers would avoid breaking the old API.

Is there something I'm missing regarding a v1 release that's important to you?

**Comment by kuritka** [comment]
Hi @samlown , I made quick review of code changes , README and new build pipes. All seems good to me . Well  done.

Except v1 stuff ,  to have some way of ensuring that there will be no breaking changes I am satisfied with everything.
